### DISCUSSION TITLE
"-depth 16" vs "PNG64:output.png" options to convert 8bit image to 16bit image

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#6118] [Help] [answered]
### ImageMagick version

7.0.8-Q16-HDRI

### Operating system, version and so on

Windows

### Description

Hello. I noticed both commands shown below convert an 8 bit image to 16bit

magick input.png -depth 16 output.png
magick input.png PNG64:output.png

As far as I can tell the output images are the same (both convert to 16bit image), but can someone confirm these two commands achieve the same thing? Or more specifically they achieve the same result as they were used in my examples?

**Comment by fmw42** [comment, marked as answer]
PNG64 with always turns on or creates an opaque transparency channel, i.e. 4 channels each at 16-bits.  -depth 16 won't enable an alpha channel if not there already.

**Reply by ghost** [reply]
Thank you.
